src/types/hooks.types.ts

```diff
@@ -0,0 +1,13 @@
+export type MessageType = 'success' | 'error' | '';
+
+export type Message = {
+  message: string;
+  type: MessageType;
+};
+
+export type UseMessageReturnType = [
```

Comment:
if you refactor the hook to return an objet, it would make this type more readable too.

I bet if any of us comes here in like 2 weeks, and we see that the first thing that is returned is a string, or the last thing is a function that might not do anything - we won't remember what's up :)